mu-plugins/blocks/global-header-footer/blocks.php

```diff
@@ -847,8 +847,10 @@ function render_global_footer( $attributes, $content, $block ) {
 	$wrapper_attributes = get_block_wrapper_attributes(
 		array( 'class' => 'global-footer wp-block-group' )
 	);
+	$tag_name = $attributes['tagName'] ?? 'footer';
```

Comment:
Do you think this should be validated somehow, maybe one of a list of possible tag names? I guess it would just be `div` or `footer`, in reality.

Reply:
Yeah that's a good idea imo.

Reply:
Added in [433b5c7](https://github.com/WordPress/wporg-mu-plugins/pull/620/commits/433b5c71c80ecadfd8eb9d8f6aa01271a17fd9ce)